src/api2/instagramv2_p.h

```diff
@@ -16,12 +19,18 @@ class Instagramv2Private: public QObject
 public:
     Instagramv2Private(Instagramv2 *q);
 
+    void loadCookies();
+
+    InstagramRequestv2 *fileRequest(QString endpoint, QString boundary, QByteArray data);
+    InstagramRequestv2 *request(QString endpoint, QByteArray post);
+
 private Q_SLOTS:
     void setUser();
     void doLogin();
     void syncFeatures();
     void profileConnect(QVariant profile);
     void configurePhoto(QVariant answer);
+    void saveCookie();
```

Comment:
```diff
-void saveCookie();
+void saveCookie() const;
```

Reply:
I tried to limit my changes to the bare minimum, and since the saveCookie() method was not const before, I didn't add const when moving it here. But I will add it.